Makefile

```diff
@@ -31,15 +31,19 @@ build: ## Build the containers
 	${DOCKER_COMPOSE} build
 
 start: ## Start the containers
-	$(MAKE) initdb
+	$(MAKE) init-db
 	${DOCKER_COMPOSE} up --detach
 
+start-db:
+	${DOCKER_COMPOSE} up --detach db
+
+init-db:
+	$(MAKE) start-db
+	${DOCKER_COMPOSE} run --rm app npm run ${INITDB}
```

Comment:
@thewilkybarkid Do we not need the `wait-healthy` anymore?

Reply:
It appears to make no difference. Hypothesis is that the database connection waits for it to be ready, but will see if we encounter a failure.

Reply:
The driver probably has a timeout on the connection; one guarantee of the `postgres` container image is that when the port is opened, the database should be ready to work (they do an elaborate boostrap where they start listening only on localhost, run setup, then stop and restart the database server to listen to the outside world).